inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -195,10 +195,8 @@
                         "type": "string"
                     },
                     "orcid": {
-                        "description": "ORCID Id when available",
                         "format": "orcid",
-                        "pattern": "\\d{4}-\\d{4}-\\d{4}-\\d{4}",
-                        "title": "ORCID ID",
```

Comment:
Mh, I've come to despise `title` and `description` that add nothing to the name of the field. But yeah, they are unrelated, so I will retract these changes.